doc/source/serve/performance.md

```diff
@@ -20,20 +19,132 @@ We are continuously benchmarking Ray Serve. The metrics we care about are latenc
 - Ray Serve’s latency overhead is single digit milliseconds, around 1-2 milliseconds on average.
 - For throughput, Serve achieves about 3-4k queries per second on a single machine (8 cores) using 1 HTTP proxy actor and 8 replicas performing noop requests.
 - It is horizontally scalable so you can add more machines to increase the overall throughput. Ray Serve is built on top of Ray,
-  so its scalability is bounded by Ray’s scalability. Please check out Ray’s [scalability envelope](https://github.com/ray-project/ray/blob/master/release/benchmarks/README.md)
+  so its scalability is bounded by Ray’s scalability. Please see Ray’s [scalability envelope](https://github.com/ray-project/ray/blob/master/release/benchmarks/README.md)
   to learn more about the maximum number of nodes and other limitations.
 
+We run long-running benchmarks nightly:
+
+```{eval-rst}
+.. list-table::
+   :header-rows: 1
+
+   * - Benchmark
+     - Description
+     - Cluster Details
+     - Performance Numbers
+   * - `Single Deployment <https://github.com/ray-project/ray/blob/master/release/serve_tests/workloads/single_deployment_1k_noop_replica.py>`_
```

Comment:
Should we specify a commit hash in these links, so these workloads can be updated without creating misleading or dead links?

Reply:
whatever we can do to prevent from dead links or 404; slightly out of date more acceptable than 404.

Reply:
Sounds good, I pinned the links using commit hashes. That should help prevent dead links or 404s.